frontend/src/pages/Overview/OverviewPage.tsx

```diff
@@ -495,8 +495,8 @@ export class OverviewPage extends React.Component<OverviewProps, State> {
       nss.push(ns.name);
     });
 
-    // @TODO add cluster
-    return Promise.all([API.getConfigValidations(nss), API.getAllIstioConfigs('', nss, [], false, '', '')])
+    // TODO: Do we need the cluster here?
```

Comment:
We need a cluster here, Istio configs loading require `cluster` param, otherwise it will take home cluster as default.

Reply:
Right. I though it was a "bulk" call... The problem I see is, the chunk is an array of mixed namespaces, so then, we should probably make a loop to make a different call per namespace. Now we are sending a call per namespace but different clusters, so it is not possible to associate them like the call it is right now, because we are also using just the namespace name, and not the whole namespace structure:

`http://localhost:3000/api/namespaces/metallb-system/metrics?filters[]=request_count&filters[]=request_error_count&duration=1800&step=180&rateInterval=180s&direction=inbound&reporter=destination`

Reply:
I think we need a separate sub-task for this, s it is out of scope of this PR.

Reply:
https://github.com/kiali/kiali/issues/6070